test/fixtures/usersegment/index.html

```diff
@@ -39,8 +39,12 @@
       keys: {
         email: 'someone@somewhere.com'
       }
-    }, function (keys) {
-      if (keys !== undefined) {
+    }, function (response) {
+      if (response &&
+          response.length &&
+          response[0].key !== undefined &&
+          Object.prototype.toString.call(response[0].values) === '[object Array]'
```

Comment:
We're not confirming attributes object is returned

Reply:
the attributes object is not always returned (depending on input values), and due to the nature of the e2e i didn't feel comfortable checking for it's presence